node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,9 +137,19 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            // Only embed text, html, images, and types that the browser has plugins to support
+            var type = widgetData.previews.targetType ? widgetData.previews.targetType.split(';')[0] : false;
+            var universalSupport = _.contains(['text/html', 'text/plain', 'image/png', 'image/jpeg', 'image/gif', 'image/svg+xml'], type);
+            var havePlugin = navigator.mimeTypes && navigator.mimeTypes[type] && navigator.mimeTypes[type].enabledPlugin;
+            var supported = widgetData.previews.targetType && (universalSupport || havePlugin);
+
             // The link can be embedded as an iFrame
-            if (embeddable) {
-                oae.api.util.template().render($('#linkpreview-iframe-template', $rootel), {'link': link}, $('#linkpreview-container'), $rootel);
+            if (embeddable && supported) {
+                if (type.substring(0, 6) === 'image/') {
+                    oae.api.util.template().render($('#linkpreview-image-template', $rootel), {'img': link, 'linkProfile': widgetData}, $('#linkpreview-container'), $rootel);
```

Comment:
I'm getting the same for that URL (or any other direct image). Webshot has been updated 22 days ago, downgrading to 0.14.0 seems to give the same results however. I'll look into it some more.